docs/architecture/products.rst

```diff
@@ -15,8 +15,18 @@ Type of cover is the only attribute which creates separate variants in our store
 Class Diagram
 -------------
 
+.. To regenerate this class digram:
+    - brew install graphviz
+    - pip3 install --install-option="--include-path=/usr/local/include/" --install-option="--library-path=/usr/local/lib/" pygraphviz
+    - pip install django-extensions
+    - INSTALLED_APPS = [
```

Comment:
are these commands bash? If no you could add `settings.` to INSTALLED_APPS.  If yes, you should probably add `export`